src/index.html

```diff
@@ -8,10 +8,206 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="POST"
+    >
+      <fieldset class="group">
+
+        <legend>Personal information</legend>
```

Comment:
Empty line should be for sibling tags besides first and last
```suggestion
      <fieldset class="group">
        <legend>Personal information</legend>
```